### PR TITLE
docs(guild): add missing features

### PR DESCRIPTION
**Pull request by MaskDuck**
## Summary

This PR documents the guild features `ANIMATED_BANNER`, `APPLICATION_COMMAND_PERMISSIONS_V2`, `INVITE_DISABLED` and `ROLE_ICONS`.

## Checklist

- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by ooliver1 — CHANGES_REQUESTED** [review]
This is missing `INVITES_DISABLED` and `ROLE_ICONS` as documented [here](https://discord.com/developers/docs/resources/guild#guild-object-guild-features).

**Review by Skelmis — CHANGES_REQUESTED** [review]

File: nextcord/types/guild.py [hunk 1]
```diff
@@ -59,12 +59,15 @@ class UnavailableGuild(TypedDict):
...
     "AUTO_MODERATION",
+    "ANIMATED_BANNER",
     "ANIMATED_ICON",
+    "APPLICATION_COMMAND_PERMISSIONS_V2",
     "BANNER",
     "COMMUNITY",
     "DEVELOPER_SUPPORT_SERVER",
     "DISCOVERABLE",
     "FEATURABLE",
+    "INVITE_DISABLED",
```
Comment (Skelmis) [inline nextcord/types/guild.py:70]:
```suggestion
    "INVITES_DISABLED",
```

**Review by Skelmis — CHANGES_REQUESTED** [review]

**Comment by ooliver1** [comment]
But their functions being documented are incorrect now.

**Review by EmreTech — APPROVED** [review]
Double checked with the [Guild Features list](https://discord.com/developers/docs/resources/guild#guild-object-guild-features).

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**